docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -163,6 +163,11 @@ cortex_querytee_responses_total{backend="<hostname>",method="<method>",route="<r
 cortex_querytee_responses_compared_total{route="<route>",result="<success|fail>"}
 ```
 
+Additionally, if backend results comparison is configured, two native-histograms are available:
+
+* `cortex_querytee_backend_request_relative_duration_seconds`: Time (in seconds) of preferred backend less secondary backend.
```

Comment:
Maybe rephrase to make it clear that it's the difference between the two?

```suggestion
* `cortex_querytee_backend_request_relative_duration_seconds`: The difference of time, in seconds, between the preferred backend request duration and the secondary backend request duration.
```

Reply:
This is copied from the help string of the new metric where the description is consistent with other metrics.